projects/components/src/file-upload/drop-zone/drop-zone.directive.ts

```diff
@@ -0,0 +1,31 @@
+import { Directive, EventEmitter, HostListener, Output } from '@angular/core';
+
+@Directive({
+  selector: '[htDropZone]'
+})
+export class DropZoneDirective {
+  @Output()
+  public readonly dropped: EventEmitter<FileList> = new EventEmitter();
+
+  @Output()
+  public readonly dragOver: EventEmitter<boolean> = new EventEmitter();
```

Comment:
When would we use dragOver output (and it's true and false value)

Reply:
This is being used to show the `blue dashed` border while dropping the files.

Reply:
see below